Makefile

```diff
@@ -45,6 +50,44 @@ validate-infra-secrets: read-keyvault-config install-fetch-config set-azure-acco
 	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -d quiet \
 		&& echo Data in ${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} looks valid
 
+.PHONY: set-space-developer
+set-space-developer: read-deployment-config ## `make dev set-space-developer USER_ID=first.last@digital.education.gov.uk`
```

Comment:
Would be cleaner to put comments on their own line rather than in the middle of the commands

Reply:
I think this allows generating help, like in apply:

```
% make
help                           Show this help
build                          Create a new image
setup                          Set up a clean database and node_modules folder for running the app or the specs in docker
stop                           bring down the containers
test                           Run the linters and specs
...
```

Reply:
That doesn't work in this repo (not sure why) :(
The syntax is also slightly different, to match apply it should be:
```
.PHONY: set-space-developer
set-space-developer:  ## `make dev set-space-developer USER_ID=first.last@digital.education.gov.uk`
   read-deployment-config
   $(if $(USER_ID), , $(error Missing environment variable "USER_ID", USER_ID required for this command to run))
   cf set-space-role $(USER_ID) dfe $(SPACE) SpaceDeveloper
```
The script that's being executed is on the line after the comments in apply

Reply:
Help now works after including help rule :)